### PR TITLE
[Term Entry] Kotlin Strings .removeRange()

### LINKED ISSUE #3734: [Term Entry] Kotlin Strings .removeRange()
### Reason/inspiration (optional)

We would like a new term entry on the `removeRange()` method for Kotlin: [Strings](https://www.codecademy.com/resources/docs/kotlin/strings). The entry should go in a new file under `docs/content/kotlin/concepts/strings/terms/removeRange/removeRange.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by 0xrahul6**
### Description

Added Term Entry for Strings .removeRange()
Closes #3734 
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/removeRange/removeRange.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`removeRange()`** method in Kotlin is used to remove a specified range of characters from a string.The removeRange() method returns a new `string` with the specified range of characters removed.
```
Comment (avdhoottt) [inline content/kotlin/concepts/strings/terms/removeRange/removeRange.md:16]:
```suggestion
The **`removeRange()`** method in Kotlin is used to remove a specified range of characters from a string. It returns a new string with the specified range of characters removed.
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@0xrahul6 Hey, Please make the changes that I've mentioned.

**Review by avdhoottt — COMMENTED** [review]
PR is ready for the next round of Review.

**Review by avdhoottt — APPROVED** [review]

**Review by SSwiniarski — APPROVED** [review]
@0xrahul6 I did one minor edit and it looks good to merge.

**Pull request merged**